.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -18,32 +22,51 @@ body:
         3.
     validations:
       required: true
+
+  # Requirements
   - type: textarea
     attributes:
       label: Requirements
       description: describe the expected behavior (eg. Fix crash that occurs when...). If left blank, the issue will likely be closed without discussion.
-      value: |
-        - eg. Fix crash that occurs when...
+      placeholder: |
```

Comment:
Let's keep this as value, I often use it as a starting point when creating bug reports.